drp/api/questions.py

```diff
@@ -0,0 +1,172 @@
+from flask import request
+from flask_restful import Resource, abort
+
+from ..db import db
+from ..models import Question, Site, Subject
+from ..swag import swag
+
+from .site import serialize_site
+from .subject import serialize_subject
+
+
+@swag.definition("Question")
+def serialize_question(question):
+    """
+    Represents a question.
+    ---
+    properties:
+      id:
+        type: integer
+      site:
+        $ref: "#/definitions/Site"
+      specialty:
+        type: string
```

Comment:
I think that their current form also includes grade.

Reply:
Good point, I missed that